go/packages/packages_linux.go

```diff
@@ -405,8 +455,19 @@ func GetInstalledPackages() (Packages, []string) {
 			pkgs.Gem = gem
 		}
 	}
+	if exists(zypper) {
```

Comment:
You don't need this check at all, zypper is just a frontend to rpm

Reply:
so you mean, we essentially will get the same package list from rpmquery and zypper query. yeah, then it makes sense to remove the blob